liquidctl/driver/hydro_platinum.py

```diff
@@ -136,20 +136,21 @@ def __init__(self, device, description, fan_count, rgb_fans, **kwargs):
             ('led', 'fixed'): 1,
             ('led', 'off'): 0,
         }
-        # the following fields are only initialized in connect()
-        self._data = None
-        self._sequence = None
 
-    def connect(self, **kwargs):
-        """Connect to the device."""
-        super().connect(**kwargs)
```

Comment:
Is there any way around this change?  Otherwise simply probing for devices will cause directories to be created for this device.

Also, why is this only a problem on this driver, and not others that use the RuntimeStorage API?

Reply:
thats a good question. Im not sure why it didn't come up with any of the other devices, I'll take another look at that.

Reply:
I moved it back to connect, the tests were only failing here before because of the sequence number part, was impacted by other tests. The new check to only set the `RuntimeStorage` if data is None will fix that behaviour for the tests so they continue to use the mock, but will not affect any real runs.